runtime/src/bank.rs

```
@@ -519,6 +519,12 @@ impl Bank {
         parents
     }
 
+    pub fn withdraw(&self, pubkey: &Pubkey, tokens: u64) {
+        let mut account = self.get_account(pubkey).unwrap_or_default();
```

If `None`, this should return an error.